csrc/anemone_mempool.c

```diff
@@ -1,11 +1,12 @@
 #include "anemone_mempool.h"
 #include <string.h>
+#include <malloc.h>
 
-// Allocate a new block with given predecessor 
+// Allocate a new block with given predecessor
 ANEMONE_STATIC
 anemone_block_t * anemone_block_create (anemone_block_t *prev, size_t num_bytes)
 {
-  void     *ptr = malloc (num_bytes);
+  void     *ptr = memalign(ANEMONE_ALIGNMENT, num_bytes);
```

Comment:
We would want to do that if we want to switch between them for testing.

From my reading of the Intel instruction manual, aligned accesses need to be 16 byte aligned, but in my testing, its fine with 8 byte alignment.